Source/AllTests.unoproj

```diff
@@ -1,4 +1,5 @@
 {
+  "BuildCondition": "UNO_TEST",
```

Comment:
Hmm, this makes it sound like uno#197 is a regression, and that similar issues might be triggered in other repos...

Reply:
The regression would be that the new version of `uno doctor` will find and build more projects. Shouldn't be a blocker if it happens though.